DnD-Montreal/session-tome
# adventures league import bug fix

## Description
Closes #172 
Fixes bug which caused backend to throw an error for certain exported characters
- added checks for different entry types
- added functionality for importing items as well as entries

## How to test
pre-existing migration was modified so first:
`php artisan migrate:fresh`

then:
`php artisan tinker`
`$user = User::factory()->create()`
`Auth::login($user)`
`$character = AdventuresLeague::getCharacter(database_path("mocks/dante.csv"))`
observe that donte is imported, a character is created, and entries are attached.

observe the last entry and note down its id, ill refer to it as k, when you run the following command replace k with the id.
`$e = Entry::find(k)`
`$e->items`
observe the two items attached to this entry
